rbd/snapshot_namespace.go

```diff
@@ -7,6 +7,11 @@ package rbd
 // #cgo LDFLAGS: -lrbd
 // #include <rbd/librbd.h>
 import "C"
+import (
```

Comment:
I think it is this project's style to leave a blank line after the `import "C"` and other imports.

Reply:
This is still a TODO